### DISCUSSION TITLE
Can we combine `useTooltipTrigger` with `useOverlay`?

### DISCUSSION THREAD

**Discussion by brandonpittman** [adobe/react-spectrum#3468] [Q&A] [answered]
I need a tooltip that renders in a `Portal`, but the standard tooltip trigger React Aria affords doesn't do that. Can we combine the tooltip hooks with the overlay ones to create a tooltip that can break out of its overflowing parents?

**Comment by snowystinger** [comment, marked as answer]
The answer to that is yes. And better yet, we do it in our Spectrum components. See https://react-spectrum.adobe.com/react-spectrum/Tooltip.html, it's rendered to the body. See our source code for how we did it, should be a pretty quick read https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/tooltip/src/TooltipTrigger.tsx 

Let us know if you run into any difficulties.

**Reply by brandonpittman** [reply]
@snowystinger Thank you very much!
